extensions/wikia/DesignSystem/scripts/DesignSystemGlobalNavigationSearchSuggestions.js

```diff
@@ -0,0 +1,86 @@
+require(
+	['jquery', 'wikia.window'],
+	function ($, window) {
+		'use strict';
+
+		var $searchInput = $('#searchInput'),
+			$searchInputWrapper = $('#searchInputWrapper'),
+			searchSuggestionsUrl = $searchInput.data('suggestions-url');
+
+		function initSuggestions() {
+			mw.loader.using('jquery.autocomplete').then(function () {
+				var autocompleteReEscape = new RegExp('(\\' + ['/', '.', '*', '+', '?', '|', '(', ')',
+						'[', ']', '{', '}', '\\'].join('|\\') + ')', 'g');
+
+				$searchInput.autocomplete({
+					serviceUrl: searchSuggestionsUrl,
+					queryParamName: $searchInput.data('suggestions-param-name'),
+					appendTo: '.wds-global-navigation__search-input-wrapper',
+					deferRequestBy: 200,
+					minLength: 3,
+					maxHeight: 1000,
+					onSelect: function (value, data, event) {
+						var valueEncoded = encodeURIComponent(value.replace(/ /g, '_')),
+							// slashes can't be urlencoded because they break routing
+							location = window.wgArticlePath.
+								replace(/\$1/, valueEncoded).
+								replace(encodeURIComponent('/'), '/');
+
+						// Respect modifier keys to allow opening in a new window (BugId:29401)
+						if (event.button === 1 || event.metaKey || event.ctrlKey) {
+							window.open(location);
+
+							// Prevents hiding the container
+							return false;
+						} else {
+							(window.Wikia.Tracker.buildTrackingFunction({
```

Comment:
Instead of calling `buildTrackingFunction` you could simply call `track()` method